Content.Server/EntityEffects/Effects/PlantMetabolism/PlantAdjustPotency.cs

```diff
@@ -1,17 +1,16 @@
 ﻿using Content.Server.Botany.Systems;
 using Content.Shared.EntityEffects;
 
-namespace Content.Server.EntityEffects.Effects.PlantMetabolism
+namespace Content.Server.EntityEffects.Effects.PlantMetabolism;
 
-{
```

Comment:
You removed the curly braces from the namespace but forgot to fix the indentation by moving everything inside to the left. Your IDE should give you warnings for not following the formatting rules.